front/src/modules/trainschedule/components/DriverTrainScheduleV2/exportDriverScheduleCSV.ts

```diff
@@ -81,12 +71,14 @@ const overloadSteps = (
     lineCode: op.line_code,
     trackName: op.track_name,
   }));
-  const speedsAtSpeedLimitChange = speedLimits.map((speedLimit) => ({
-    position: mmToM(speedLimit.position),
-    speed: interpolateValue(trainRegime, speedLimit.position, 'speeds'),
-    speedLimit: speedLimit.speed,
-    time: interpolateValue(trainRegime, speedLimit.position, 'times'),
-  }));
+  const speedsAtSpeedLimitChange = speedLimits.speeds.map((_speed, index) => {
+    const boundary = index > 0 ? speedLimits.internal_boundaries[index - 1] : 0;
+    return {
+      position: boundary,
```

Comment:
In the previous version, the position was in meter, but here I think you are giving it in mm